src/Scroll-Bar.scss

```diff
@@ -0,0 +1,34 @@
+@import 'style/constants';
+
+.MissionScroll{
+
+    height:200px;
+
+    .card{
+        display: flex;
+        background-color: #2c3e50;
+        min-width: 100%;
+        min-height: auto;
+        overflow-x: auto; 
+        padding: 10px 0;
+    }
+
+    .card::-webkit-scrollbar {
```

Comment:
I suspect that this isn't going to work on lots of browsers. Have you tested that?

Probably a question for the PM.
Properly hiding the scrollbar is a bit of a pain, and often not desirable (especially on desktop)

Can be hidden with JS, but would probably be worth a ticket on its own.